src/pages/login/LoginForm.tsx

```diff
@@ -1,8 +1,46 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
 import Photo from '../../images/Rectangle.svg';
 
 const LoginForm: React.FC = () => {
-   
+
+  const [ username , setUsername ] = useState<string>('');
+  const [ userpassword, setUserpassword ] = useState<string>('');
+  const [ getdata, setGetdata ] = useState<string[]>([]);
+
+  useEffect(() => {
+    fetch('http://tiknikstyle.com/wp-json/wp/v2/home_page_texts')
+      .then(res => res.json())
+      .then((data) => setGetdata(data));
+    },[setGetdata])
+    console.log(getdata);
+  
+  const  formHendler = async (event: React.MouseEvent<HTMLButtonElement>) => {
+
+    setUsername('');
+    setUserpassword('');
+
+    if(username && userpassword) {
+      try {
```

Comment:
here I see async and await

in useEffect I see .then and .catch

lets use one single way.


let move with try()catch() and async/await way